test/unit/test_threaded_downloader.py

```diff
@@ -347,7 +347,8 @@ def connection_error(*args, **kwargs):
 
             self.assertEqual(session.get.call_count, 2)
 
-            expected_log_msg = ['Download of http://pulpproject.org/primary.xml failed. Re-trying.',
```

Comment:
I didn't remove any just moved the old first line down so they are in order that that are logged.